courses/free-courses-en.md

```diff
@@ -156,6 +156,7 @@
 
 ### C
 
+* [C language tutorials](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - Code with harry 
```

Comment:
- Playlist is called: `C Language Tutorials In Hindi` and creator channel `CodeWithHarry`
- Complete platform: YouTube

```suggestion
* [C Language Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - 
CodeWithHarry (YouTube)
```

Reply:
Youtube is an exception to the platform name guideline - don't need to add it; As @davorpa notes, please use "C Language Tutorials In Hindi" as the title